include/h2o/memory.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 #define H2O_STRUCT_FROM_MEMBER(s, m, p) ((s *)((char *)(p)-offsetof(s, m)))
 #define H2O_ALIGNOF(type) (__alignof__(type))
 
-#if __GNUC__ >= 3
+#if (__GNUC__ >= 3) || defined(__clang__)
```

Comment:
Cousin you please revert the change? We can and do rely at other parts of the source code on clang setting __GNUC__.

Other changes look fine to me.

Reply:
yes, reverted.